### ISSUE TITLE
Some minor mouse problems: shutter/lag in menus/briefings, clics don't recognized in briefings and mouse wheel don't recognized

### ISSUE THREAD

**Issue by Sol1vaN** [InsanityBringer/InjectD3#3]
Hi,

Excelent work with this great patch.
Is the first time that I can play descent 3 in some comfortable way.

However, I know there's a some things to improve.
In this case, the mouse shuttering are terrible on menus.
The other problem, like say the title, is mouse wheel don't recognized when you want to bind some functions.
And the clic action is not recognized by briefings menus, but the clics are well recognized in the main menu, is strange.
The only way to navigate in the briefings menus is with the arrow keys from the keyboard, for exit you need to press ESC.

Here's a video showing how mouse works in game vs main menu vs briegings menu.
https://www.youtube.com/watch?v=r_mSHRQXVfA

All this problems are solved by selecting OFF the "New mouse library", btw, you lost all the fixes for mouse movements.

Well, I hope that you can fix this some day.
Cheers.

**Comment by InsanityBringer** [comment]
I believe what may be contributing to the stuttering issue is that Descent 3 caps the game framerate to 20 while in the menus, making it feel pretty rough. I was playing around with moving this cap. I'm also sorry about the mouse not working in briefings, I've tried unsuccessfully to track down this bug, but it's probably something obvious right under my nose. The mouse system is still being tuned, but Descent 3's mouse code in general is pretty cursed. 

I'll be working on a fix for the mousewheel not being read soon. I had overlooked that the game accepts it, but it won't be too hard to support.
